ocrd_tesserocr/recognize.py

```diff
@@ -347,7 +347,7 @@ def _process_existing_glyphs(self, tessapi, glyphs, word_image, word_xywh):
                 glyph.set_TextEquiv([])
             #glyph_text = tessapi.GetUTF8Text().rstrip("\n\f")
             glyph_conf = tessapi.AllWordConfidences()
-            glyph_conf = glyph_conf[0]/100.0 if glyph_conf else 0.0
+            glyph_conf = glyph_conf[0]/100.0 if glyph_conf else 1.0
```

Comment:
Both defaults make sense but the new interpretation seems more intuitive.

Reply:
Sorry, this was unrelated. I had stumbled across that earlier and left it uncommitted.

1 is neutral in this context...